packages/at_chops/lib/src/at_chops_base.dart

```diff
@@ -108,4 +110,8 @@ abstract class AtChops {
   /// Create a hash of input [signedData] using a [hashingAlgorithm].
   /// Refer to [DefaultHash] for default implementation of hashing.
   String hash(Uint8List signedData, AtHashingAlgorithm hashingAlgorithm);
+
+  /// Reads a public key from SIM card based on ID. SIM card should have an IOTSafe compliant app
```

Comment:
From SIM is only one example of what an implementation could do.

Reply:
please check whether my latest docs are fine.